src/coordinator/ts/m3db/types.go

```diff
@@ -37,7 +37,8 @@ type SeriesBlock struct {
 // SeriesBlocks is a placeholder until it is added to M3DB
 type SeriesBlocks struct {
```

Comment:
Could be useful to have a small description of what's in a SeriesBlock, since it's a pretty complicated object